src/System Application/App/Performance Profiler/src/PerfProfileActivityType.enum.al

```diff
@@ -0,0 +1,38 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.Tooling;
+
+/// <summary>
+/// The supported activity types for creating scheduled based sampling profiles
+/// </summary>
+enum 1932 "Perf. Profile Activity Type"
+{
+    Access = Public;
+    Extensible = false;
+
+    /// <summary>
+    /// The web client activity type
+    /// </summary>
+    value(0; WebClient)
```

Comment:
Can we separate these words with a space? "Web Client" and same for the api client "Web API Client"

Reply:
Done